hidimstat/stat_tools.py

```diff
@@ -66,21 +66,22 @@ def pval_from_scale(beta, scale, testing_sign, eps=1e-14):
     scale : ndarray, shape (n_features,)
         Value of the standard deviation of the parameters.
 
-    testing_sign : {'minus', 'plus'}
-        If 'minus', testing for negativity (low p-values means we may
-        reject negativity). If 'plus', testing for positivity (low p-values
-        means we may reject positivity).
+    testing_sign : {'plus', 'minus'}
```

Comment:
Not sure what you want to do.
Normally a user could want to get a one-sided p-value or a two-sided one.
If he wants to have p-values for the opposite statistic, he tests the opposite hypothesis.